genai/generativelanguagepb_veneer.gen.go

```diff
@@ -605,8 +605,8 @@ func (v HarmProbability) String() string {
 	return fmt.Sprintf("HarmProbability(%d)", v)
 }
 
-// Model is information about a Generative Language Model.
-type Model struct {
+// ModelInfo is information about a Generative Language Model.
```

Comment:
Maybe just "about a Language model"?

"Generative" maybe applies less to embeddings models

Reply:
Done.